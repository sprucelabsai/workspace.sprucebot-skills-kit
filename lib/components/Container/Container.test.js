'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

require('jest-styled-components');

var _Container = require('./Container');

var _Container2 = _interopRequireDefault(_Container);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('Container Component', function () {
	test('it renders', function () {
		// 	const tree = renderer
		// 		.create(<Container>Container children</Container>)
		// 		.toJSON()
		// 	expect(tree).toMatchSnapshot()
	});
});