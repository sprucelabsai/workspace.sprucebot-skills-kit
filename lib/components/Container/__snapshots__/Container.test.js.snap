// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component it renders 1`] = `
<div
  className="container "
>
  Container children
</div>
`;
